rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -252,7 +252,7 @@ _precompiled_apple_resource_bundle = rule(
         ),
         bundle_extension = attr.string(
             mandatory = False,
-            default = "bundle",
+            default = ".bundle",
```

Comment:
Without the leading period this call to `paths.replace_extension` in rules_apple fails to remove the extension:
```
    bundle_name_with_extension = bundle_name + bundle_extension
    product_name = paths.replace_extension(bundle_name_with_extension, "")

    # Values for string replacement substitutions to perform in the merged
    # Info.plist
    substitutions = {
        "BUNDLE_NAME": bundle_name_with_extension,
        "PRODUCT_NAME": product_name,
    }
```